lib/index.ts

```diff
@@ -25,7 +25,7 @@ export const getJWTToken = (opts: JWTOptions): Promise<JWTResponse> => {
     let audience = opts.audience || 'https://login.salesforce.com';
     var options: jwt.SignOptions = {
 		issuer: opts.clientId,
-		audience,
+		audience: audienceTranslator(audience),
```

Comment:
I almost wonder if it would make more sense to just expose `audienceTranslator` as a separate export so it could then be used explicitly.

EG: 

```
import { audienceTranslator, getJWTToken } from 'salesforce-jwt-promise';

getJWTToken({
        audience: audienceTranslator('https://my-custom-domain.salesforce.com'),
        clientId: clientId,
        privateKey: privateKey,
        userName: 'user@toImpersonate.com',
})
```

Reply:
Your original properties combines audience with the authUrl...there's only one point of entry, so I split them out from the inside (see line38, which needs to use the actual audience, but line 28 which can't).

So keeping it "inside" lets it be used without affecting any code calling (non-breaking change).

Reply:
I supposed you could also add a 5th optional property (instanceurl) and keep audience only allowing login/test, and leave figuring that out (and making sure the combo is valid) to the user.

Reply:
ohhh I see. Sorry I misunderstood at first.  

I think the best way to do this would be too just add an additional *optional* parameter of `instanceUrl`, which if not set would be defaulted to `audience` (to maintain backwards compatibility).

```
export interface JWTOptions {
    clientId: string,
    privateKey: string;
    userName: string;
    audience?: string; //defaults to login.salesforce.com
    instanceUrl?: string; //defaults to audience
}
```

I'm not completely _opposed_ to the translate function, but my goal is to keep this library as simple and easy to understand as possible.

Thoughts?